<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sortwindow</class>
 <widget class="QDialog" name="sortwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sort Choices</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pics.qrc">
    <normaloff>:/Imagenes/BookSeller-Logo.png</normaloff>:/Imagenes/BookSeller-Logo.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-2</y>
     <width>321</width>
     <height>205</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pics.qrc">:/Imagenes/Hoja.JPG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>261</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QRadioButton" name="TitleRB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="AuthorRB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Author</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="YearRB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="EditorialRB">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Editorial</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="TypeRB">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
